src/kwsbin/transcripts-to-fsts.cc

```diff
@@ -46,21 +60,34 @@ int main(int argc, char *argv[]) {
 
```

Comment:
In the usage message at line 51, need angle brackets around command line args, and need to indicate the optional weights-rspecifier using
        `[<weights-rspecifier>]`
there should probably be some explanation there of what the weights are.  
Actually I am not really happy about it being called "weights-rspecifier" if they actually represent
the minus-log of the weight, because "weights" on Kaldi command lines tend to refer to things
that should be >= 0, i.e. actual weights.  I'd prefer to see it called "costs-rspecifier" and the
variable be called a "cost".